feature/staticroute/static_route_support/README.md

```diff
@@ -0,0 +1,114 @@
+# RT-1.26: Basic static route support
+
+## Summary
+
+- Static route ECMP must be supported
+- Static route metric must be supported
+- Static route Administrative Distance / Preference must be supported
+- ```set-tag``` attribute must be supported for static routes
+- Disabling recursive nexthop resolution must be supported
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed
+
+## Procedure
+
+#### Initial Setup:
+
+*   Connect DUT port-1, port-2 and port-3 to ATE port-1, port-2 and port-3 respectively
+*   Configure IPv4/IPv6 addresses on DUT and ATE the interfaces
+*   Configure one IPv4 destination i.e. ipv4-network connected to ATE
+*   Configure one IPv6 destination i.e. ipv6-network connected to ATE
+
+### RT-1.26.1 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route ECMP
+
+*   Configure IPv4 static routes:
+    *   Configure one IPv4 static route i.e. ipv4-route-a on the DUT for destination ipv4-network with the next hop of port-1
+    *   Configure another IPv4 static route i.e. ipv4-route-b on the DUT for destination ipv4-network with the next hop of port-2
+*   Configure IPv6 static routes:
+    *   Configure one IPv6 static route i.e. ipv6-route-a on the DUT for destination ipv6-network with the next hop of port-1
+    *   Configure another IPv6 static route i.e. ipv6-route-b on the DUT for destination ipv6-network with the next hop of port-2
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is received from DUT on both port-1 and port-2 and ECMP works
+
+### RT-1.26.2 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route metric
+
+*   Configure metric of ipv4-route-b and ipv6-route-b to 1000
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/metric
+*   Validate that the metric is set correctly
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/state/metric
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is received from DUT on port-1 only
+
+### RT-1.26.3 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route prefrence 
+
+*   Configure preference of ipv4-route-a and ipv6-route-a to 1000
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/preference
+*   Validate that the preference is set correctly
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/state/preference
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is now received from DUT on port-2 only
```

Comment:
Shouldn't this use ipv(4,6)-network-a?

Reply:
No, ipv[4,6]-network-a's metric is lower than ipv[4,6]-network-b but because preference of ipv[4,6]-network-a is 1000 it will not be used. (Ah, 1000 probably is illegal, the value ranges from (0 - 255), will change it to 200